src/domelts.js

```diff
@@ -0,0 +1,72 @@
+const getDate = () => {
+  const currentDate = document.querySelector('#date');
+  currentDate.textContent = new Date().toUTCString();
+};
+
+async function getWeather(value = 'Bamenda') {
+  const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${value}&APPID=a65bf913c63cc981687c4ea80470725d`, { mode: 'cors' });
+  const weatherdata = await response.json();
```

Comment:
- [ ] Your application can benefit from an error handling for `promises` or `async/await`, like a try-catch block. You can give feedback for the user, like 'location not found'
![Screenshot from 2020-04-20 21-20-09](https://user-images.githubusercontent.com/30977845/79811605-be332f00-834c-11ea-8157-0a5999dc8f09.png)